stylesheet/style.css

```
@@ -0,0 +1,1042 @@
+	background-color: #393d50;
+	color: #fff;
+	text-transform: uppercase;
+	font-weight: bold;
+	margin: 18px 0 30px 0;
+	border-radius: 30px;
+	padding: 6px 25px;
+	letter-spacing: 2;
+}
+
+.btn-style-one i {
+	padding-right: 5px;
+}
+
+.btn-style-one:hover {
+	color: #fff;
+}
+
+.column {
+	transition: transform .3s;
+}
+
+.column:hover {
+	transform: scale(1.1);
+	cursor: pointer;
+	z-index: 10;
+}
+
+footer {
+	background-color: #212331;
+	padding: 100px 0 0 0;
+	color: var(--grey);
+	font-size: 15px;
+}
+
+.footer__box {
+	padding-bottom: 50px;
+}
+
+.footer__box__title {
+	color: #fff;
+	text-transform: uppercase;
+	font-size: 19px;
+	font-weight: 1000;
+	padding-bottom: 30px;
+}
+
+.footer__box__title:after {
+	content: '';
+	margin-left: 10px;
+	width: 8px;
+	height: 8px;
+	display: inline-block;
+	background-image: url(../images/Untitled-2.png);
+}
+
+.address {
+	padding-bottom: 35px;
+	margin-bottom: 35px;
+	border-bottom: 1px dotted
+}
+
+.contact-box {
+	padding: 15px 0;
+}
+
+.contact-box a {
+	color: var(--lightcolor);
+}
+
+.address i,
+.contact-box i {
+	padding-right: 10px;
+}
+
+.post-box {
+	padding: 15px 0;
+	border-bottom: 1px dotted
+}
+
+.post-box:last-child {
+	border-bottom: 0;
+}
+
+.post-box img {
+    width: 70px;
+    height: 70px;
+	border-radius: 50%;
+	float: left;
+	margin-right: 10px;
+}
+
+.post-box a {
+	color: var(--grey);
+}
+
+.post-box a:hover {
+	color: var(--lightcolor);
+}
+
+.text-blue {
+	color: var(--lightcolor);
+}
+
+.post-box>p {
+	padding-top: 5px;
+}
+
+.comments {
+	float: right;
+}
+
+.footer__box__link ul {
+	padding-left: 0;
+	margin-left: 15px;
+	list-style: disc;
+}
+
+.footer__box__link ul li {
+	padding: 10px 0;
+	border-bottom: 1px dotted
+}
+
+.footer__box__link ul li:last-child {
+	border-bottom: 0;
+}
+
+.footer__box__link ul li a {
+	color: #fff;
+}
+
+.footer__box__link ul li a:hover {
+	color: var(--lightcolor);
+}
+
+.information img {
+	max-width: 70px;
+	height: auto;
+	float: left;
+}
+
+.logo-footer-name {
+	color: #fff;
+	font-size: 35px;
+	text-transform: uppercase;
+	margin-top: -10px;
+	line-height: 35px;
+	letter-spacing: 3;
+}
+
+.information p {
+	padding-top: 40px;
+}
+
+.footer__box input,
+.footer__box textarea {
+	background-color: #181828;
+	border-color: #394050;
+	padding-left: 20px;
+}
+
+.footer__box .form-group {
+	position: relative;
+}
+
+.footer__box .form-group i {
+	position: absolute;
+	top: 35%;
+	right: 10px;
+}
+
+.footer__box input {
+	height: 45px;
+}
+
+.footer__box textarea {
+	width: 100%;
+	height: 100px;
+}
+
+.footer__box .form-group {
+	margin-top: 12px;
+	margin-bottom: 20px;
+}
+
+.footer__box .btn-submit {
+	margin-top: 22px;
+	width: 100%;
+	text-transform: uppercase;
+	font-weight: bold;
+	background-color: #394050;
+}
+
+.footer__box .btn-submit:hover {
+	background-color: var(--lightcolor);
+	color: #fff;
+}
+
+.form-newsletter {
+	margin-top: 30px;
+	position: relative;
+}
+
+.form-newsletter i {
+	position: absolute;
+	top: 0%;

```

```suggestion
	top: 0;

```
e dùng % thì right cũng dùng % theo, ko dùng thì cả 2 đều k dùng, viết code sao cho đồng bộ với nhau chứ đừng để lệch tone như này